pkgs/development/libraries/openal-soft/default.nix

```diff
@@ -18,7 +18,9 @@ stdenv.mkDerivation rec {
     sha256 = "051k5fy8pk4fd9ha3qaqcv08xwbks09xl5qs4ijqq2qz5xaghhd3";
   };
 
-  buildInputs = [ cmake ]
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = []
```

Comment:
Agreed, but I wrote it that way for stylistic consistency with the NIX_LDFLAGS definition further down. I could get rid of it anyway, should I?